Dockerfile.centos.7

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2018-2019, Intel Corporation
+#
+# 'recipe' for Docker to build an RPM
+#
+
+# Pull base image
+FROM centos:7
+MAINTAINER daos-stack <daos@daos.groups.io>
+
+# use same UID as host and default value of 1000 if not specified
+ARG UID=1000
+
+# Update distribution
+#Nothing to do for CentOS
+
+# Install basic tools
+RUN yum install -y epel-release
+RUN yum install -y mock make rpm-build curl createrepo rpmlint redhat-lsb-core \
+                   git
+
+# Add build user (to keep rpmbuild happy)
+ENV USER build
+ENV PASSWD build
+RUN useradd -u $UID -ms /bin/bash $USER
+RUN echo "$USER:$PASSWD" | chpasswd
+# add the user to the mock group so it can run mock
+RUN usermod -a -G mock $USER
+
+# mock in Docker needs to use the old-chroot option
+RUN grep use_nspawn || \
+    echo "config_opts['use_nspawn'] = False" >> /etc/mock/site-defaults.cfg
+
```

Comment:
Uneeded whitespace at the end.  Only worth resolving if you need to push for other issues.

Reply:
Don't need to push for other issues.

Reply:
Just a reminder of the whitespace here in case any of my other new comments result in a new commit being pushed.